domain/src/test/java/org/oppia/domain/topic/StoryProgressControllerTest.kt

```diff
@@ -527,22 +525,30 @@ class StoryProgressControllerTest {
 
   private fun verifyTopicProgressFailed() {
     verify(mockTopicProgressObserver, atLeastOnce()).onChanged(topicProgressResultCaptor.capture())
-    assertThat(topicProgressResultCaptor.value.isSuccess()).isFalse()
+    val topicProgress = topicProgressResultCaptor.value.getOrThrow()
+    assertThat(topicProgress.storyProgressMap.size).isEqualTo(0)
   }
 
   private fun verifyTopicProgressSucceeded() {
     verify(mockTopicProgressObserver, atLeastOnce()).onChanged(topicProgressResultCaptor.capture())
     assertThat(topicProgressResultCaptor.value.isSuccess()).isTrue()
   }
 
+  private fun verifyStoryProgressFailed(){
```

Comment:
Nit on spacing: extra space needed between ``)`` and ``{``.

Reply:
Fixed this nit.